sway/ui-sample.cfg

```diff
@@ -0,0 +1,38 @@
+#
+# Colors
+#
+# Property Name         Border     BG        Text    Indicator  Child Border
+client.focused          $bg3       $bg2      $fg1    $bg2       $bg2
+client.focused_inactive $bg1       $bg0      $bg3    $bg0       $bg0
+client.unfocused        $bg1       $bg0      $bg3    $bg0       $bg0
```

Comment:
for Child Border, I would use:

$dark-green
$bg2
$bg2
$dark-red

It's somewhat difficult to see the border on floating terminals with the current settings.